test/test-init.js

```diff
@@ -1,6 +1,6 @@
 const { parseEther } = ethers.utils;
 
-const PROXY_CREATION = "ProxyCreation";
+// const PROXY_CREATION = "ProxyCreation";
```

Comment:
Hey, feel free to remove them, not just comment, we have previous repo with all this data, so we're not going to lose it

Reply:
Overall it's good, I'll approve once you clear up commented code